tlatools/org.lamport.tlatools/src/tla2sany/parser/Operators.java

```diff
@@ -148,4 +122,213 @@ public static void printTable() {
+    new Operator("\\propto",      50,  50,  Operators.assocNone, Operators.infix),
+  };
+  
+  /*
+   * Different ways of writing the same operator. The zeroeth item is the
+   * canonical way.
+   */
+  private static final String[][] OperatorSynonyms = new String[][] {
+    new String[] {"\\lnot", "~", "\\neg", "¬"},
+    new String[] {"[]", "□"},
+    new String[] {"<>", "◇"},
+    new String[] {"\\land", "/\\", "∧"},
+    new String[] {"\\lor", "\\/", "∨"},
+    new String[] {"\\equiv", "<=>", "≡", "⇔"},
+    new String[] {"/=", "#", "≠"},
+    new String[] {"\\leq", "<=", "=<", "≤"},
+    new String[] {"\\geq", ">=", "≥"},
+    new String[] {"\\times", "\\X", "×"},
+    new String[] {"\\intersect", "\\cap", "∩"},
+    new String[] {"\\union", "\\cup", "∪"},
+    new String[] {"\\o", "\\circ", "∘"},
+    new String[] {"\\oplus", "(+)", "⊕"},
+    new String[] {"\\ominus", "(-)", "⊖"},
+    new String[] {"\\odot", "(.)", "⊙"},
+    new String[] {"\\oslash", "(/)", "⊘"},
+    new String[] {"\\otimes", "(\\X)", "⊗"},
+    new String[] {"\\approx", "≈"},
+    new String[] {":=", "≔"},
+    new String[] {"\\asymp", "≍"},
+    new String[] {"\\bigcirc", "◯"},
+    new String[] {"::=", "⩴"},
+    new String[] {"\\bullet", "●"},
+    new String[] {"\\cdot", "⋅"},
+    new String[] {"\\cong", "≅"},
+    new String[] {"\\div", "÷"},
+    new String[] {"\\doteq", "≐"},
+    new String[] {"..", "‥"},
+    new String[] {"...", "…"},
+    new String[] {"!!", "‼"},
+    new String[] {"\\gg", "≫"},
+    new String[] {"=>", "⇒"},
+    new String[] {"\\in", "∈"},
+    new String[] {"=|", "⫤"},
+    new String[] {"~>", "↝"},
+    new String[] {"\\ll", "≪"},
+    new String[] {"-|", "⊣"},
+    new String[] {"\\notin", "∉"},
+    new String[] {"-+->", "⇸"},
+    new String[] {"\\prec", "≺"},
+    new String[] {"\\preceq", "⪯"},
+    new String[] {"\\propto", "∝"},
+    new String[] {"??", "⁇"},
+    // This synonym was disabled in the parser so disabling it here too
+    //new String[] {"%", "\mod"},
+    new String[] {"|=", "⊨"},
+    new String[] {"|-", "⊢"},
+    new String[] {"\\sim", "∼"},
+    new String[] {"\\simeq", "≃"},
+    new String[] {"\\sqcap", "⊓"},
+    new String[] {"\\sqcup", "⊔"},
+    new String[] {"\\sqsubset", "⊏"},
+    new String[] {"\\sqsubseteq", "⊑"},
+    new String[] {"\\sqsupset", "⊐"},
+    new String[] {"\\sqsupseteq", "⊒"},
+    new String[] {"\\star", "⋆"},
+    new String[] {"\\subset", "⊂"},
+    new String[] {"\\subseteq", "⊆"},
+    new String[] {"\\succ", "≻"},
+    new String[] {"\\succeq", "⪰"},
+    new String[] {"\\supset", "⊃"},
+    new String[] {"\\supseteq", "⊇"},
+    new String[] {"\\uplus", "⊎"},
+    new String[] {"||", "‖"},
+    new String[] {"\\wr", "≀"},
+    new String[] {"^+", "⁺"},
+  };
+
+  /**
+   * Statically initializes all operators in the definition table. This will
+   * be executed when this class loads into memory.
+   */
+  static {
```

Comment:
It was moved since it needs to be below the new static fields. I also changed the exception to a RuntimeException to match the built-in op initialization code.